src/include/rear/ui.rb

```diff
@@ -103,16 +119,22 @@ def GetMountpoints
       mountpoints.map {|e| e + "/*" }
     end
 
-    # helper to compare an array with a default array and returns a formatted
-    # list containing the items that need to be added to the array to become
-    # equal to the default array.
-    def ArrayChecker(value, default)
-      to_add = default - value
+    # Helper to compare two arrays. The action can be either :add or :remove. The default array
+    # specifies which items are expected (:add), or not expected (:remove) to be in the value array.
+    # The returned value is a list containing items that are missing and should be added (:add), or
+    # are not desired and should be removed (:remove).
+    def ArrayChecker(value, default, action)
```

Comment:
please document also parameters using yardoc `@param` tags.

Reply:
Added `@param` tags to this method and reworded its description to avoid duplicating some information that is now described in the tags.